code/game/gamemodes/cult/cult_mode.dm

```diff
@@ -27,7 +27,7 @@ GLOBAL_LIST_EMPTY(all_cults)
 		return FALSE
 	if(iscultist(mind.current))
 		return TRUE //If they're already in the cult, assume they are convertable
-	if(HAS_TRAIT(mind, TRAIT_HOLY))
+	if(HAS_MIND_TRAIT(mind.current, TRAIT_HOLY))
```

Comment:
It also works on minds like this
```suggestion
	if(HAS_MIND_TRAIT(mind, TRAIT_HOLY))
```

Reply:
This fails to compiles,as it checks mind.mind, due to the helper.
Its the reason i changed it to this.

Reply:
Then just change `mind` to `src`?

Reply:
`src` in this case does not have `status_traits` as its an undefined variable
The helper would also read this as `src.mind` which is also an undefined variable ontop of the fact that `src.mind.status_traits` does not exist.
This is due to how the proc this check is in is `/proc/is_convertable_to_cult(datum/mind/mind)` and `current` point back to the mob again.

Reply:
Whoops sorry, I was looking at the wrong proc for this, yeah I agree `mind.current` is needed here